### ISSUE TITLE
Accuracy alone is Misleading

### ISSUE THREAD

**Issue by Atomic-Joy** [PalakRay07/Heart-Disease-Detection-with-Machine-Learning#1]
Currently:

```
accuracy_score
```

Problem:
If dataset is slightly imbalanced : accuracy lies

Add:
```
from sklearn.metrics import roc_auc_score
roc_auc_score(y_test, rf_pred)
```
